internal/resultstore/resultstore.go

```diff
@@ -149,3 +149,31 @@ func (rs *ResultStore) Save(ctx context.Context, p Pkg, analysis interface{}) er
 
 	return nil
 }
+
+// MakeFilename returns the default filename to use for saving analysis results,
+// using an optional label.
+// If the package has a version, the default filename is
+// "<label>-<version>.json" if label is nonempty, or <version>.json otherwise.
+// If the package does not have a version specified, the default filename is
+// "<label>.json" if label is nonempty, or "results.json" if not.
+func MakeFilename(p Pkg, label string) string {
+	prefix := "results"
+
+	version := p.Version()
+	if version != "" && label != "" {
+		prefix = label + "-" + version
+	} else if version != "" && label == "" {
```

Comment:
Nit: some of these conditions are a bit redundant. 

e.g. the two elses can just be 


```go
} else if version != "" {
} else if label != "" {
}
```

Reply:
Yeah I thought making it more explicit would be more readable at first but I like your way more